components/modal/modal.css

```diff
@@ -0,0 +1,42 @@
+@import url(/css/base.css);
+@import url(/css/utility.css);
+@import url(/documentation/documentation.css);
+
+/* modal starts here */
+.et_modal-container {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+}
+
+.et_modal-container h3 {
+    color: var(--secondary-color);
+    font-size: 1.2rem;
+    text-align: center;
+}
+
+/* simple-modal starts here */
+.modal-container {
+    align-items: center;
+    background-color: var(--white-color);
+    border: 1px solid var(--light-gray);
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    padding-bottom: 1rem;
+    margin: 1rem 0rem;
+    width: 100%;
+}
```

Comment:
if possible you can remove  align-items: center; display: flex; flex-direction: column; from modal-container and
add et_modal-container class where ever you have used class modal-container  as these three properties are same 
just a suggestion or can create one utility class as named flex-col-center which having these three properties

Reply:
yes, was thinking about the same, but instead of providing and changing the classes name, i would better go with utility classes. Thanks for the suggestion though, will definitely look into it, and will refactor my code a liitle more!!